### DISCUSSION TITLE
Is second heating circuit supported ?

### DISCUSSION THREAD

**Discussion by FlorianLaunay** [liudger/python-bsblan#831] [Q&A]
Hi @liudger !

Thanks for this great library! My question is in the title :wink: .
Does it support a second heating circuit?

I found #59 but after reading your code I am not so sure now.

Thank you in advance for your reply.

**Comment by liudger** [comment]
> Hi @liudger !
> 
> 
> 
> Thanks for this great library! My question is in the title :wink: .
> 
> Does it support a second heating circuit?
> 
> 
> 
> I found #59 but after reading your code I am not so sure now.
> 
> 
> 
> Thank you in advance for your reply.

Yes in the upcoming home-assistant release 2026.5.0 multiple circuits should be supported!
